ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js

```diff
@@ -392,11 +405,18 @@ module.exports = function CesiumMap(
         })
       })
     },
+    timeoutId: NaN,
     onCameraMoveStart(callback) {
+      clearTimeout(this.timeoutId)
       map.scene.camera.moveStart.addEventListener(callback)
     },
     onCameraMoveEnd(callback) {
-      map.scene.camera.moveEnd.addEventListener(callback)
+      const timeoutCallback = () => {
+        this.timeoutId = setTimeout(() => {
+          callback()
+        }, 300)
+      }
+      map.scene.camera.moveEnd.addEventListener(timeoutCallback)
```

Comment:
I think this timeout should be up to who is calling onCameraMoveEnd.  But it's just 300 ms, so 🤷